newsletter.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
since newsletter.html is autogenerated, can we remove this file from the commit? and perhaps even add the `newsletter.html` file to the `.gitignore`

Reply:
I was planning to delete it before my last push.